Automattic/vaultpress
# General: move VaultPress class to own file & add PHP53 shortcircuit

Fixes #15

- Moving the main VaultPress class to its own class allows us to move all PHP 5.3+ code out of the plugin's entry point, vaultpress.php
- In that plugin's entry point, we now check the PHP version in use on the site, and pause the VaultPress plugin (i.e. stop loading it) if you are running an old version of PHP.

![image](https://user-images.githubusercontent.com/426388/59356915-27f4ac80-8d2a-11e9-9160-ae9d9b03f5fd.png)


**To test**

- Try activating the plugin on a site running PHP 5.2. It should be paused.
- Try activating the plugin on a site running a more modern version of PHP; nothing should change.
